### PR TITLE
Add custom color support for Daily Tasks

### PR DESCRIPTION
**Pull request by Hydrox6**
Also, fix the notifications not working with the transparent chat

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java [hunk 1]
```diff
@@ -63,11 +64,33 @@ default boolean showStaves()
...
+
+	@ConfigItem(
+		keyName = "dailyTaskColorTransparent",
+		name = "Color (Transparent Chat)",
+		description = "Choose what color the daily task messages will be in the transparent chatbox",
+		position = 5
+	)
+	default Color dailyTaskColorTransparent()
+	{
+		return new Color(255, 0, 0);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java]: Use Color.RED

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java [hunk 2]
```diff
@@ -63,11 +64,33 @@ default boolean showStaves()
...
+
+	@ConfigItem(
+		keyName = "dailyTaskColor",
+		name = "Color",
+		description = "Choose what color the daily task messages will be",
+		position = 4
+	)
+	default Color dailyTaskColor()
+	{
+		return new Color(255, 0, 0);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java]: Use Color.RED

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java [hunk 3]
```diff
@@ -145,7 +148,8 @@ private boolean checkCanCollectEssence()
 
 	private void cacheColors()
 	{
-		chatMessageManager.cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, Color.RED, false), ChatMessageType.GAME).refreshAll();
+		chatMessageManager.cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, config.dailyTaskColor(), false), ChatMessageType.GAME).refreshAll();
+		chatMessageManager.cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, config.dailyTaskColorTransparent(), true), ChatMessageType.GAME).refreshAll();
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java]: Call refreshAll only once

**Comment by Hydrox6** [comment]
Implemented with #3421

**Pull request closed without merging**